src/services/connector/handlers/sinkMmdlData.js

```diff
@@ -0,0 +1,56 @@
+import * as dynamodb from "../../../libs/dynamodb-lib";
+
+async function myHandler(event, _context, _callback) {
+  console.log("Received event:", JSON.stringify(event, null, 2));
+
+  const recordValueObject = JSON.parse(event.value);
+
+  // Typically the PROGRAM_TYPE_CODE will match this _transNbr key
+  //   MAC: "mac179_transNbr",
+  //   CHP: "chp179_transNbr",
+  //   HHS: "hhs_transNbr",
+
+  // But the PROGRAM_TYPE_CODE might not match the expected transmittal number.
+  // As such we should use whichever _transNbr is available in the record
+  const possibleTransmittalNumberKeys = Object.keys(recordValueObject).filter(
+    (k) =>
+      k === "mac179_transNbr" || k === "chp179_transNbr" || k === "hhs_transNbr"
+  );
+
+  if (possibleTransmittalNumberKeys.length === 0) {
+    console.log("No transmittal id available for the record", event);
+    return;
+  }
+
+  if (possibleTransmittalNumberKeys.length > 1) {
+    console.log(
+      "Error - Multiple transmittal ids available for the record",
+      event
+    );
+    console.log(possibleTransmittalNumberKeys);
```

Comment:
Should the error logging message say that it is going to output the event object next and then the possibleTransmittalNumberKeys object?  And should those two be combined into one call of console.error?